aio/content/examples/schematics-for-libraries/projects/my-lib/package.json

```
@@ -0,0 +1,20 @@
+// #docplaster
+// #docregion collection
+{
+  "name": "my-lib",
+  "version": "0.0.1",
+// #enddocregion collection
+  "scripts": {
+    "build": "../../node_modules/.bin/tsc -p tsconfig.schematics.json",
+    "copy:schemas": "cp --parents schematics/*/schema.json ../../dist/my-lib/",
+    "copy:files": "cp --parents -p schematics/*/files/** ../../dist/my-lib/",
+    "copy:collection": "cp schematics/collection.json ../../dist/my-lib/schematics/collection.json",
```

You can remove the `collection.json` part from the destination place to copy to.

```suggestion
    "copy:collection": "cp schematics/collection.json ../../dist/my-lib/schematics",
```

---

This will be resolved in #29505